.maintain/config/bridge-pangoro-goerli.toml

```diff
@@ -7,11 +25,10 @@ outbound_address                 = "0xEe8CA1000c0310afF74BA0D71a99EC02650798E5"
 chain_message_committer_address  = "0x01D1Cb6D629536b893f2343B95124aDB6a8ae68b"
 lane_message_committer_address   = "0x5313Bf259a4C723cdD6bE4B6387d21321f180D03"
 account                          = "0x..."
+# private_key should not start with 0x
 private_key                      = "//Alice"
```

Comment:
Rename this private key value is better because this is evm account's private key